### PR TITLE
Tweak README for Cloud Firestore

### LINKED ISSUE #153: Remove optional callbacks from docs for Cloud Firestore?
Hi! Noticed that optional callbacks (e.g. readyCallback) do not function in 3.0.0-alpha.1 for Cloud Firestore. As @posva noted in [this comment](https://github.com/posva/vuexfire/issues/117#issuecomment-365183139), `bindFirebaseRef` returns a promise so we can check things that way.

Should we remove the optional callbacks section from the README in that case? I went in a few circles trying to get it to work today :) LMK @posva and I can do a PR.

### PR DESCRIPTION
**Pull request by bsaf**
I think this might be my first PR so feel free to change or reject!

Fixes #153.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,26 +113,13 @@ You can check out the examples by opening the html files in your browser, or che
 This object contains VuexFire internal mutations. They are all prefixed by
 `vuexfire/`. This object must be added in the root Store mutations object.
 
-### bindFirebaseRef(key, ref[, options])
+### bindFirebaseRef(key, ref)
 
 _Only available inside of an enhanced action_
 
 Binds a firebase reference to a property in the state. If there was already
 another reference bound to the same property, it unbinds it first.
```
Comment (posva) [inline README.md]: Could we add some explanation about it returning a promise that succeeds once the data is ready or that throws if anything goes wrong?

Comment (bsaf) [reply README.md]: Sure :) I updated the commit.

Comment (bsaf) [reply README.md]: PS: Quick question... I squashed locally and forced pushed my update – next time should I leave the commits separate and let you squash on your side?

Comment (posva) [reply README.md]: don't worry about it next time, I squash PRs 🙂

**Pull request merged**